homeassistant/components/zwave_js/cover.py

```diff
@@ -79,17 +79,19 @@ async def async_set_cover_position(self, **kwargs: Any) -> None:
 
     async def async_open_cover(self, **kwargs: Any) -> None:
         """Open the cover."""
-        target_value = self.get_zwave_value("Open")
-        await self.info.node.async_set_value(target_value, PRESS_BUTTON)
+        target_value = self.get_zwave_value("targetValue")
+        await self.info.node.async_set_value(target_value, 99)
 
     async def async_close_cover(self, **kwargs: Any) -> None:
         """Close cover."""
-        target_value = self.get_zwave_value("Close")
-        await self.info.node.async_set_value(target_value, PRESS_BUTTON)
+        target_value = self.get_zwave_value("targetValue")
+        await self.info.node.async_set_value(target_value, 0)
 
     async def async_stop_cover(self, **kwargs: Any) -> None:
         """Stop cover."""
-        target_value = self.get_zwave_value("Open")
-        await self.info.node.async_set_value(target_value, RELEASE_BUTTON)
-        target_value = self.get_zwave_value("Close")
-        await self.info.node.async_set_value(target_value, RELEASE_BUTTON)
+        target_value = self.get_zwave_value("Open") or self.get_zwave_value("Up")
+        if target_value:
+            await self.info.node.async_set_value(target_value, RELEASE_BUTTON)
```

Comment:
I think these RELEASE_BUTTON and PRESS_BUTTON names are confusing, and probably a carryover from OZW. node-zwave-js doesn't have a "button" concept. Can we just call these STOP (RELEASE) and START (PRESS) instead, or just use True and False? This matches the zwave-js [implementation](https://github.com/zwave-js/node-zwave-js/blob/023cb39a1c72eaa3827f48ca16bc867f0778db4a/packages/zwave-js/src/lib/commandclass/MultilevelSwitchCC.ts#L348) better.

Reply:
Those are easily changed. I didn't alter what they were as I wasn't sure if there was some more significant meaning behind them.

Reply:
I removed the PUSH_BUTTON and changed RELEASE_BUTTON to just be sending False